README.md

```diff
@@ -0,0 +1,19 @@
+![Are you mocking me?](https://fakeiteasy.github.io/img/fakeiteasy_logo_256.png)
+
+[![NuGet version](https://img.shields.io/nuget/v/FakeItEasy.Analyzers.svg?style=flat)](https://www.nuget.org/packages/FakeItEasy)
+[![Build status](https://ci.appveyor.com/api/projects/status/tmxobysgprwpecsb/branch/master?svg=true)](https://ci.appveyor.com/project/FakeItEasy/fakeiteasy-analyzers/branch/master)
+
+Roslyn analyzers to detect incorrect usages of FakeItEasy.
+
+Add an analyzer to your projects to avoid, detect, and fix common programming errors when working with FakeItEasy.
+Analyzers exist for both C# and VisualBasic.
+* [Documentation](https://fakeiteasyanalyzers.readthedocs.io/en/stable/)
+* [Chat](https://gitter.im/FakeItEasy/FakeItEasy)

```

Comment:
_should_ we advertise a FakeItEasy/FakeItEasy.Analyzers chat?

Reply:
No, I think we should use the main chat. No point in having a separate one.